appium/options/ios/xcuitest/webview/webview_options.py

```diff
@@ -0,0 +1,533 @@
+    - Set
+        - `self.safari_log_all_communication` = `value`
+    
+    Parameters
+    ----------
+    `value`: `bool`
+
+    Returns
+    -------
+    - Get
+        - `Optional[bool]`
+    - Set
+        - `None`
+    """
+
+
+class SafariOpenLinksInBackgroundOption(SupportsCapabilities):
+    SAFARI_OPEN_LINKS_IN_BACKGROUND = "safariOpenLinksInBackground"
+    safari_open_links_in_background = OptionsDescriptor[Optional[bool], bool]
+    (SAFARI_OPEN_LINKS_IN_BACKGROUND)
+    """
+    Whether Safari should allow links to open in new windows.
+    Default keeps current sim setting.
+
+    Usage
+    -----
+    - Get
+        - `self.safari_open_links_in_background`
+    - Set
+        - `self.safari_open_links_in_background` = `value`
+    
+    Parameters
+    ----------
+    `value`: `bool`
+
+    Returns
+    -------
+    - Get
+        - `Optional[bool]`
+    - Set
+        - `None`
+    """
+
+
+class SafariSocketChunkSizeOption(SupportsCapabilities):
+    SAFARI_SOCKET_CHUNK_SIZE = "safariSocketChunkSize"
+    safari_socket_chunk_size = OptionsDescriptor[Optional[int], int]
+    (SAFARI_SOCKET_CHUNK_SIZE)
+    """
+    The size, in bytes, of the data to be sent to the Web Inspector on
+    iOS 11+ real devices. Some devices hang when sending large amounts of
+    data to the Web Inspector, and breaking them into smaller parts can be
+    helpful in those cases. Defaults to 16384 (also the maximum possible).
+
+    Usage
+    -----
+    - Get
+        - `self.safari_socket_chunk_size`
+    - Set
+        - `self.safari_socket_chunk_size` = `value`
+    
+    Parameters
+    ----------
+    `value`: `int`
+
+    Returns
+    -------
+    - Get
+        - `Optional[int]`
+    - Set
+        - `None`
+    """
+
+
+class SafariWebInspectorMaxFrameLengthOption(SupportsCapabilities):
+    SAFARI_WEB_INSPECTOR_MAX_FRAME_LENGTH = "safariWebInspectorMaxFrameLength"
+    safari_web_inspector_max_frame_length = OptionsDescriptor[Optional[int], int]
+    (SAFARI_WEB_INSPECTOR_MAX_FRAME_LENGTH)
+    """
+    The maximum size in bytes of a single data frame for the Web Inspector.
+    Too high values could introduce slowness and/or memory leaks.
+    Too low values could introduce possible buffer overflow exceptions.
+    Defaults to 20MiB (20*1024*1024).
+
+    Usage
+    -----
+    - Get
+        - `self.safari_web_inspector_max_frame_length`
+    - Set
+        - `self.safari_web_inspector_max_frame_length` = `value`
+    
+    Parameters
+    ----------
+    `value`: `int`
+
+    Returns
+    -------
+    - Get
+        - `Optional[int]`
+    - Set
+        - `None`
+    """
+
+
+class WebkitResponseTimeoutOption(SupportsCapabilities):
+    WEBKIT_RESPONSE_TIMEOUT = "webkitResponseTimeout"
+    webkit_response_timeout = OptionsDescriptor[Optional[timedelta], Union[timedelta, int]]
+    (WEBKIT_RESPONSE_TIMEOUT, transform_duration_get, transform_duration_set)
+    """
+    Time to wait for a response from WebKit in a Safari session.
```

Comment:
the docstring seems to be cut off. What is the measurement unit and the default value?